core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -107,7 +107,7 @@ export class RteHelperModalComponent {
   @Input() customizationArgSpecs: CustomizationArgsSpecsType;
   @Input() attrsCustomizationArgsDict: CustomizationArgsForRteType;
   @Input() componentIsNewlyCreated: boolean;
-  modalIsLoading: boolean = true;
+  modalIsLoading: boolean = false;
```

Comment:
It is not clear to me how this fixes the issue. Can you please explain?

Reply:
I think you might be close to a solution. IIRC the setTimeout is introduced to let Angular know when to trigger change detection, it can have a 0 delay -- we don't actually want the code to execute at a later time. Could it be that the logic to focus on the first input is triggered at an incorrect place? Perhaps we should trigger it after a modal opens and fully renders. It could be that the focus service looks for the input and tries to focus on it when it is not actually rendered / visible on the screen.

Reply:
@kevintab95 Is the current solution not correct?

Reply:
I'm not convinced that setting the flag to a default value should be the correct fix for this -- we haven't identified what causes the focus to trigger and when it does it. Besides, setting the "modal is loading" flag to a false value at the start indicates that the modal is fully visible when it might not be.

I think you're right about the focus being affected by the status of the modal but we kind of need to understand why that is. e.g. identify the method that triggers the focus in the code.